app/polymer/src/client/manager/provider.js

```diff
@@ -0,0 +1,51 @@
+import { location } from 'global';
+import qs from 'qs';
+import React from 'react';
+import { Provider } from '@storybook/ui';
+import addons from '@storybook/addons';
+import createChannel from '@storybook/channel-postmessage';
+import Preview from './preview';
+
+export default class ReactProvider extends Provider {
```

Comment:
Is this called ReactProvider and not PolymerProvider because it bridging between Polymer and React?